### ISSUE TITLE
Spec.address: Invalid value: []string(nil): one of address or selector should be specified

### ISSUE THREAD

**Issue by wh211212** [chaos-mesh/chaos-mesh#4277]
## Bug Report

**What version of Kubernetes are you using?**

1.24
**What version of Chaos Mesh are you using?**

2.6.1
**What did you do? / Minimal Reproducible Example**

physical-machine-chaos  stress-cpu
**What did you expect to see?**
https://chaos-mesh.org/zh/docs/simulate-physical-machine-chaos/ 

![image](https://github.com/chaos-mesh/chaos-mesh/assets/16891171/abef56d0-e5fc-4953-88c5-05d69ad41b5b)

error.api.internal_server_error: admission webhook "vphysicalmachinechaos.kb.io" denied the request: Spec.address: Invalid value: []string(nil): one of address or selector should be specified

**What did you see instead?**
none
**Output of chaosctl**

        2023-12-06T07:13:40.299Z        DEBUG   controller-runtime.webhook.webhooks     admission/http.go:96    received request        {"webhook": "/validate-chaos-mesh-org-v1alpha1-physicalmachinechaos", "UID": "59a0cf78-9410-42b6-a8b6-afb912e190c1", "kind": "chaos-mesh.org/v1alpha1, Kind=PhysicalMachineChaos", "resource": {"group":"chaos-mesh.org","version":"v1alpha1","resource":"physicalmachinechaos"}}
        2023-12-06T07:13:40.299Z        INFO    PhysicalMachineChaos-resource   v1alpha1/zz_generated.chaosmesh.go:1534 validate create {"name": "pm-cpu-load"}
        2023-12-06T07:13:40.300Z        DEBUG   controller-runtime.webhook.webhooks     admission/http.go:143   wrote response  {"webhook": "/validate-chaos-mesh-org-v1alpha1-physicalmachinechaos", "code": 403, "reason": "Spec.address: Invalid value: []string(nil): one of address or selector should be specified", "UID": "59a0cf78-9410-42b6-a8b6-afb912e190c1", "allowed": false}
        2023-12-06T07:13:40.300Z        DEBUG   controller-runtime.webhook.webhooks     admission/http.go:96    received request        {"webhook": "/validate-auth", "UID": "cc145501-9bc8-4249-8f26-5cfc7cf5476b", "kind": "chaos-mesh.org/v1alpha1, Kind=PhysicalMachineChaos", "resource": {"group":"chaos-mesh.org","version":"v1alpha1","resource":"physicalmachinechaos"}}
        2023-12-06T07:13:40.300Z        DEBUG   controller-runtime.webhook.webhooks     admission/http.go:143   wrote response  {"webhook": "/validate-auth", "code": 200, "reason": "", "UID": "cc145501-9bc8-4249-8f26-5cfc7cf5476b", "allowed": true}
        2023-12-06T07:23:29.890Z        DEBUG   controller-runtime.webhook.webhooks     admission/http.go:96    received request        {"webhook": "/mutate-chaos-mesh-org-v1alpha1-physicalmachinechaos", "UID": "989781a0-2728-4a80-a593-d95bb3e64f2b", "kind": "chaos-mesh.org/v1alpha1, Kind=PhysicalMachineChaos", "resource": {"group":"chaos-mesh.org","version":"v1alpha1","resource":"physicalmachinechaos"}}
        2023-12-06T07:23:29.891Z        DEBUG   controller-runtime.webhook.webhooks     admission/http.go:143   wrote response  {"webhook": "/mutate-chaos-mesh-org-v1alpha1-physicalmachinechaos", "code": 200, "reason": "", "UID": "989781a0-2728-4a80-a593-d95bb3e64f2b", "allowed": true}
        2023-12-06T07:23:29.898Z        DEBUG   controller-runtime.webhook.webhooks     admission/http.go:96    received request        {"webhook": "/validate-chaos-mesh-org-v1alpha1-physicalmachinechaos", "UID": "c4de5f88-9fd5-4d55-8c10-d3ab2a05c207", "kind": "chaos-mesh.org/v1alpha1, Kind=PhysicalMachineChaos", "resource": {"group":"chaos-mesh.org","version":"v1alpha1","resource":"physicalmachinechaos"}}
        2023-12-06T07:23:29.898Z        INFO    PhysicalMachineChaos-resource   v1alpha1/zz_generated.chaosmesh.go:1534 validate create {"name": "pm-cpu-load"}
        2023-12-06T07:23:29.899Z        DEBUG   controller-runtime.webhook.webhooks     admission/http.go:143   wrote response  {"webhook": "/validate-chaos-mesh-org-v1alpha1-physicalmachinechaos", "code": 403, "reason": "Spec.address: Invalid value: []string(nil): one of address or selector should be specified", "UID": "c4de5f88-9fd5-4d55-8c10-d3ab2a05c207", "allowed": false}
        2023-12-06T07:23:29.901Z        DEBUG   controller-runtime.webhook.webhooks     admission/http.go:96    received request        {"webhook": "/validate-auth", "UID": "c6ba88d0-d2fd-40be-94ed-5efd498ea134", "kind": "chaos-mesh.org/v1alpha1, Kind=PhysicalMachineChaos", "resource": {"group":"chaos-mesh.org","version":"v1alpha1","resource":"physicalmachinechaos"}}
        2023-12-06T07:23:29.901Z        DEBUG   controller-runtime.webhook.webhooks     admission/http.go:143   wrote response  {"webhook": "/validate-auth", "code": 200, "reason": "", "UID": "c6ba88d0-d2fd-40be-94ed-5efd498ea134", "allowed": true}

**Comment by STRRL** [comment]
Do you already registed physical machine to Chaos Mesh, and already selected it as the target of the PhysicalMachineChaos?

**Comment by wh211212** [comment]
> Do you already registed physical machine to Chaos Mesh, and already selected it as the target of the PhysicalMachineChaos?

yes,  apply yaml can running physical machine chaos ,but in dashboard can‘t not create experiment
![image](https://github.com/chaos-mesh/chaos-mesh/assets/16891171/b9b336fd-bf34-4ff8-bbb5-edfa508328f7)

**Comment by wh211212** [comment]
> Do you already registed physical machine to Chaos Mesh, and already selected it as the target of the PhysicalMachineChaos?

worker node chaosd logs
![image](https://github.com/chaos-mesh/chaos-mesh/assets/16891171/f466ad9e-a80e-4ad6-b210-5dc668ca0a41)
